.travis.yml

```
@@ -17,6 +17,7 @@ env:
   - CRATE=boards/trinket_m0 FEATURES="--features=unproven"
   - CRATE=boards/samd11_bare FEATURES="--features=unproven" BUILDMODE="--release"
   - CRATE=boards/samd21_mini FEATURES="--features=unproven"
+  - CRATE=boards/arduino_mkr1000 EXAMPLES="--example=blinky_basic" FEATURES="--features=unproven,usb"
```

```suggestion
  - CRATE=boards/arduino_mkr1000 FEATURES="--features=unproven,usb"
```
What you have is correct, but this change will ensure that any examples added in the future are built as well without needing to update the Travis config.